denops_std/function/types.ts

```diff
@@ -1,8 +1,33 @@
+import { ensure, isLike } from "../deps.ts";
+
 export type ScreenPos = {
   row: number;
   col: number;
   endcol: number;
   curscol: number;
 };
 
-export type Position = [bufnum: number, lnum: number, col: number, off: number];
+const mockScreenPos: ScreenPos = {
```

Comment:
may

It's more like "A reference structure of ScreenPos" thus I prefer "refScreenPos" or a similar name